releases/release-1.18/release_team.md

```diff
@@ -2,7 +2,7 @@
 
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
-| Lead |  Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) |  |
+| Lead | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@@mrbobbytables`)  |
```

Comment:
```suggestion
| Lead | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`)  |
```

Still has the double `@`.

Reply:
ha didn't see that bit.
ptal 
/cc @justaugustus